app/views/pages/helpdesk_request_submitted.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :page_title, 'Information Received' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-6">
+      <h1 class="govuk-panel__title">
+        Information received
+      </h1>
+      <div class="govuk-panel__body">
+        We’ll send your TRN by email or get in touch to ask for more information
+      </div>
+    </div>
+
+    <h2 class="govuk-heading-m">What happens next?</h2>
+    <p class='govuk-body'>
+      You’ll get an email with your TRN if we find a match. Otherwise, we’ll be in touch to ask for more information.
+    </p>
+    <p class='govuk-body'>
+      We aim to respond in 5 working days.
+    </p>
+
+    <h2 class="govuk-heading-m">For urgent requests, call the Teaching Regulation Agency</h2>
+
+    <p class="govuk-body">
+      Give the helpdesk your request number: <%= @trn_request&.id %>
+    </p>
```

Comment:
Does this need a check to make sure the id is present, and if not don't show the whole paragraph?

Reply:
I think rather there shouldn't be any optional chaining here; the controller should redirect to the start page if a user tries to reach this page without a `@trn_request`... The page should blow up if `id` isn't defined.

WDYT @felixclack?

Reply:
Yeah, makes sense to me.

Reply:
I thought sometimes the Zendesk ticket number sometimes wouldn't be available, and that's ok?